Documentation/EmacsConfiguration.md

```diff
@@ -31,7 +29,7 @@ that is needed by `clangd`.
   :hook ((c++-mode) . lsp-deferred)
   :commands lsp
   :config
-  (setq lsp-clients-clangd-args '("-j=4" "-background-index" "--log=error" "--clang-tidy" "--enable-config"))
```

Comment:
@sin-ack  surely this isn't valid emacs LSP syntax, how do I reference the project folder?

Reply:
Dropped the emacs config from the PR, chatting on discord it sounds like it will need some help to set serenity-specific clangd options.